Extensions/better_reblogs.js

```diff
@@ -201,65 +219,85 @@ XKit.extensions.better_reblogs = new Object({
 
 	run_flat: function() {
 
-		var list_sel = "";
+		const reblogTop = XKit.extensions.better_reblogs.reblogs_class;
+		const reblogHeader = XKit.extensions.better_reblogs.reblog_headers_class;
+		const reblogAvWrap = XKit.extensions.better_reblogs.reblog_av_wrappers_class;
+		const reblogUrlWrap = XKit.extensions.better_reblogs.reblog_url_wrappers_class;
+		const reblogUrl = XKit.extensions.better_reblogs.reblog_urls_class;
+		const rows = XKit.extensions.better_reblogs.rows_class;
+		const txtBlk = XKit.extensions.better_reblogs.text_blocks_class;
+		var list_sel = reblogTop + ">div:nth-of-type(2)";
 		if (this.preferences.remove_last_user.value) {
-			XKit.tools.add_css(".reblog-list-item.contributed-content .reblog-header {display: none;}", "better_reblogs");
-			list_sel = ".reblog-list ";
+			XKit.tools.add_css("div[data-is-contributed-content]>" + reblogHeader + " {display: none;}", "better_reblogs");
```

Comment:
keyToCss returns a comma separated list of classes, so this will turn into the css string
```css
div[data-is-contributed-content] > .class1, .class2 {
  display: none;
}
```

Instead, you'll want to do something like:

```js
XKit.css_map.keyToClasses("reblogHeader", cls => `[data-is-contributed-content] > ${cls}`).join(", ")
```

Reply:
I did forget to do this in a couple cases, but reblogHeader and others only return a single class. As far as I can tell, the number of list items is constant. Is it our convention to do it anyway?

Reply:
Yes, it's good to be future-proofed

Reply:
The problem is that while the list of classes is constant within a single release, Tumblr could add a new class named reblogHeader at any given time, and depending on compilation order/etc, that could massively break things. Just seems like it's safer to err on the side of caution here.